verbit/streaming_client.py

```diff
@@ -92,7 +111,7 @@ def max_connection_retry_seconds(self, val: float):
     def start_stream(self,
                      media_generator,
                      media_config: MediaConfig = None,
-                     response_types: ResponseType = ResponseType.Transcript) -> typing.Generator:
+                     response_types: ResponseType = ResponseType.Transcript) -> typing.Generator[typing.Dict, None, None]:
```

Comment:
It's type hinting of the generator (syntax:  [yield_type, send_type, return_type] )
 a generator yielding 'dictionaries' not consuming `.send()` not implementing a return-type
 
The PEP says it's also common to hint them as `typing.Iterator[Dict]` instead, since a non-`.send()` generator, 'behaves like an iterator'